mingw-w64-args/PKGBUILD

```
@@ -0,0 +1,50 @@
+# Maintainer: llm96 <llm96@fervidex.net>
+
+_realname=args
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=6.4.6
+pkgrel=1
+pkgdesc="Simple header-only C++ argument parser library (mingw-w64)"
+arch=('any')
+mingw_arch=('mingw64' 'ucrt64' 'clang64' 'clangarm64')
+msys2_references=(
+  'aur: args'
+)
+url="https://github.com/Taywee/${_realname}"
```

```suggestion
url="https://github.com/Taywee/args"
```
please, don't use variables inside url. it's convenient to have a clickable url inside source